index.d.ts

```diff
@@ -86,6 +86,11 @@ export type TrackParams = {
   timestamp?: Date;
 } & IdentityOptions;
 
+export type TrackMessage<PropertiesType> = Omit<TrackParams, 'event' | 'properties'> & {
+  event?: string;
+  properties: PropertiesType;
+} & Record<string, any>;
+
```

Comment:
Here we are making the event as optional parameter in track call. What is the motivation for this change?

Reply:
I'm surprised this change was made. I've removed the whole type as it is not needed.